Mathlib/Tactic/MoveAdd.lean

```diff
@@ -223,10 +223,11 @@ If `exs` is the list `[e₁, e₂, ..., eₙ]` of `Expr`essions, then `sumList p
 `prepOp (prepOp( ... prepOp (prepOp e₁ e₂) e₃) ... eₙ)`.
 -/
 partial
-def sumList (prepOp : Expr) : List Expr → Expr
```

Comment:
Can you document this new boolean argument?

Reply:
(i'd be Inclined to move it outside the pattern match so that you can name it.

Reply:
Documentation added and also I pulled out the new `Bool`ean argument out of the pattern match.